src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__title--big">
+      <div
+        class="search__link--big"
+        data-qa="big"
```

Comment:
The `div` element on this line is missing a closing tag. Ensure that all `div` elements are properly closed to maintain valid HTML structure.